app/assets/stylesheets/games.scss

```diff
@@ -20,6 +20,10 @@ body {
   text-align:center;
   display: table-cell;
   vertical-align:middle;
+  &:active {
+    background-color: #99ff8c;
+    opacity: 0.6;
+  }
```

Comment:
This highlights a link that is selected. What we are trying to do here is something a bit different. We want the link to route to the piece#show action. Then the square that is selected will be highlighted on the new page. I can explain better when we talk on hangouts.